benchmark/utils.py

```diff
@@ -0,0 +1,123 @@
+import io
+import math
+import random
+import numpy as np
+from collections import defaultdict
+
+
+
+corpus = [
+    "The quick brown fox jumps over the lazy dog.",
+    "Artificial intelligence is transforming the world.",
+    "Natural language processing is a field of artificial intelligence.",
+    "Machine learning models require a lot of data.",
+    "Deep learning is a subset of machine learning.",
+    "Supervised learning relies on labeled data for training.",
+    "Unsupervised learning can discover hidden patterns in data.",
+    "Reinforcement learning involves agents learning from their environment.",
+    "Neural networks are inspired by the human brain.",
+    "Convolutional neural networks are effective for image recognition.",
+    "Recurrent neural networks are suitable for sequence data.",
+    "Transfer learning allows models to leverage pre-trained knowledge.",
+    "Hyperparameter tuning is crucial for optimizing model performance.",
+    "Gradient descent is an optimization algorithm used in training.",
+    "Overfitting occurs when a model performs well on training data but poorly on new data.",
+    "Cross-validation helps to evaluate model performance more reliably.",
+    "Data augmentation techniques can improve model robustness.",
+    "The Turing Test assesses a machine's ability to exhibit intelligent behavior.",
+    "Ethics in AI is a critical area of research and development.",
+    "Generative adversarial networks can create realistic synthetic data."
+]
```

Comment:
These are all still relatively the same length of text. I recommend using a dataset from huggingface

Reply:
Fixed